packages/itwin/measure-tools/src/ui-2.0/MeasureToolsUiProvider.tsx

```diff
@@ -64,6 +67,10 @@ export class MeasureToolsUiItemsProvider implements UiItemsProvider {
       }
 
       if (toolbarOrientation === ToolbarOrientation.Vertical) {
+        // don't display measurement tools on a sheet view
+        const vp = IModelApp.viewManager.selectedView;
+        if (vp && ViewHelper.isSheetView(vp))
+          return [];
         return [
           ToolbarItemUtilities.createGroupButton(
```

Comment:
the last parameter for createGroupButton is overrides, here you could define a isHidden value that you set with a ConditionalBooleanValue that syncs with the ViewStateChange and checks if the view is a sheet